### PR TITLE
sda-3086 Check that we have something to iterate over, before we iterate

### PR DESCRIPTION
**Pull request by johankwarnmarksymphony**
https://perzoinc.atlassian.net/browse/sda-3086 CLONE - SDA - App is crashed if user quite app while sharing screen

### CODE REVIEW

File: src/renderer/notification.ts [hunk 1]
```diff
@@ -409,23 +409,30 @@ class Notification extends NotificationHandler {
...
     };
     const inactiveNotificationWindows = { ...[], ...this.inactiveWindows };
-    for (const activeWindow of activeNotificationWindows) {
-      if (activeWindow && windowExists(activeWindow)) {
-        await this.hideNotification(
-          (activeWindow as ICustomBrowserWindow).clientId,
-        );
+    if (activeNotificationWindows.length > 0) {
+      for (const activeWindow of activeNotificationWindows) {
+        logger.info('notification: qqq');
```
Comment (psjostrom) [inline src/renderer/notification.ts:422]: qqq?

File: src/renderer/notification.ts [hunk 2]
```diff
@@ -409,23 +409,30 @@ class Notification extends NotificationHandler {
...
+          );
+        }
       }
     }
-    for (const inactiveWindow of inactiveNotificationWindows) {
-      if (inactiveWindow && windowExists(inactiveWindow)) {
-        inactiveWindow.close();
+    if (inactiveNotificationWindows.length > 0) {
+      for (const inactiveWindow of inactiveNotificationWindows) {
+        logger.info('notification: wwww');
```
Comment (psjostrom) [inline src/renderer/notification.ts:432]: wwww?

File: src/renderer/notification.ts (hunk 1 above)
Comment (KiranNiranjan) [inline src/renderer/notification.ts:422]: Need this log here?

File: src/renderer/notification.ts (hunk 2 above)
Comment (KiranNiranjan) [inline src/renderer/notification.ts:432]: Same here

**Review by psjostrom — APPROVED** [review]
Could use better logging :) Otherwise good.

**Review by KiranNiranjan — APPROVED** [review]

File: src/renderer/notification.ts (hunk 1 above)
Comment (johankwarnmarksymphony) [reply src/renderer/notification.ts:422]: To quick, I will remove it

File: src/renderer/notification.ts (hunk 2 above)
Comment (johankwarnmarksymphony) [reply src/renderer/notification.ts:432]: To quick, I will remove it

**Review by VishwasShashidhar — APPROVED** [review]

**Pull request closed without merging**